plantswap/src/app/[lang]/planten/page.js

```diff
@@ -3,13 +3,24 @@ import Title from "../../../../components/Title/Title.js";
 import Header from "../../../../components/Header/Header.js";
 import Layout from "../../../../components/Layout/Base.js";
 import { getDictionary } from "../../../../get-dictionary.js";
+import dataset from "../swap/data.json";
+import PlantSwitcher from "../../../../components/Switcher/Switcher.js";
+import SwapFlow from "../../../../components/SwapFlow/SwapFlow.js";
+
+// console.log(dataset);
 
 export default async function Home({ params }) {
   const dictionary = await getDictionary(params.lang);
   return (
-    <Layout locale={params.lang} dictionary={dictionary}>
+    <Layout locale={params.lang} dictionary={dictionary} noFooter={true}>
       <main>
-        <Title title={"h1"}>Hier komt een overzicht van alle planten</Title>
+        {/* <Title title={"h1"}>Hier komt een overzicht van alle planten</Title> */}
```

Comment:
Found some commented- out code here. To keep our codebase neat and maintainable, it's often best to remove these.